docs/src/main/sphinx/admin/properties-catalog.md

```diff
@@ -0,0 +1,42 @@
+# Catalog management properties
+
+(prop-catalog-management)=
+
+## `catalog.management`
+
+- **Type:** {ref}`prop-type-string`
+- **Allowed values:** `static`, `dynamic`
+- **Default value:** `static`
+
+Trino reads catalog property files and configures available catalogs on server
+startup. When set to `dynamic`, catalog configuration can also be managed using
+[](/sql/create-catalog) and [](/sql/drop-catalog). Creating and dropping
+catalogs using these SQL commands will create new and delete existing catalog
+property files.
+
+:::{warning}
+This feature is experimental only. Because of the security implications the
+syntax might change and be backward incompatible.
+:::
+
+:::{warning}
+Some connectors are known not to release all resources when dropping a catalog
+that uses such connector. This includes all connectors that can read data from
+HDFS, which are {doc}`/connector/hive`, {doc}`/connector/iceberg`,
+{doc}`/connector/delta-lake`, and {doc}`/connector/hudi`.
+:::
+
+:::{warning}
+The complete CREATE CATALOG query can be logged, and visible in the {doc}`Web
+UI </admin/web-interface>`. This includes any sensitive properties, like
+passwords and other credentials. See {doc}`secrets </security/secrets>`.
+:::
+
+## `catalog.prune.update-interval`
```

Comment:
I just noticed I missed `catalog.prune.enabled` but I'm not sure what's the use case for disabling it.

Reply:
I'm not sure why I added that.  Maybe it made testing easier.  Anyway, I'm not sure we need to document the enable